Library/ViewModels/LandingPageViewModelTests.swift

```diff
@@ -0,0 +1,18 @@
+@testable import Library
+import Prelude
+import ReactiveExtensions_TestHelpers
+import ReactiveSwift
+import UIKit
+import XCTest
+
+internal final class LandingPageViewModelTests: TestCase {
+  fileprivate let vm: LandingPageViewModelType = LandingPageViewModel()
+
+  fileprivate let landingPageCards = TestObserver<[UIView], Never>()
```

Comment:
I also don't think we should emit actual `UIView`s here, rather the data that will be used to configure the views or an `enum` type representing them.

Reply:
Yes. sorry, I forgot to mention that this was also temporary. On the second PR (containing the cards, the signal emits `[LandingPageCardType]`). So for now, I will delete the property from the view model test 👍